lib/CMakeLists.txt

```diff
@@ -343,6 +338,12 @@ else()
 endif()
 
 if(RUBY_VENDORDIR)
+    file(RELATIVE_PATH LIBFACTER_INSTALL_RELATIVE ${RUBY_VENDORDIR} ${CMAKE_INSTALL_PREFIX})
```

Comment:
We didn't other places, so I was hoping not. I'll try a run installing to "C:/Program Files" just to be sure.